pkgs/tools/networking/goflow/default.nix

```diff
@@ -0,0 +1,26 @@
+{ buildGoModule
+, fetchFromGitHub
+, lib
+}:
+
+buildGoModule rec {
+  pname = "goflow";
+  version = "3.4.2";
```

Comment:
The latest version is 3.4.3, see the Git tag.

Reply:
yep. Just changed the version :D

Reply:
Squash the changes and we're good to go.

Reply:
squashed and edited the PR :D